src/page/product-list.page.ts

```diff
@@ -4,7 +4,7 @@ export class ProductListPage {
   private addToCartButton: ElementFinder;
 
   constructor () {
-    this.addToCartButton = $('#center_column a.button.ajax_add_to_cart_button.btn.btn-default');
+    this.addToCartButton = $('.ajax_add_to_cart_button > span');
```

Comment:
try to find a better locator, something that uses the name or title attribute for example, in that way, we can understand what element you are referring